File-Formats/DocIO/Word-document/Iterating-Word-document-elements.md

```diff
@@ -1059,4 +1059,8 @@ private static void IterateParagraph(ParagraphItemCollection paraItems)
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-document/Iterate-document-elements).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-document/Iterate-document-elements).
+
+T> Passing null for both the property names and property values in above APIs, you can also find an item in a Word document without relying on any property.
+
+T> If you wish to find an item in Word document rather than iterating through each element one by one, you can use finding the item functionality to achieve it.
```

Comment:
Please add hyperlink "[finding the item functionality](https://help.syncfusion.com/find-item-in-word-document)"

Reply:
Addressed